ui/app/AppLayouts/Wallet/controls/AssetDelegate.qml

```diff
@@ -58,7 +59,8 @@ Item {
     StyledText {
         id: assetCurrencyBalance
         color: Style.current.secondaryText
-        text: Utils.toLocaleString(currencyBalance.toFixed(2), localAppSettings.locale) + " " + assetDelegate.currency.toUpperCase()
+        //TODO figure out what goes here
+        text: "%"
```

Comment:
It's unclear to me why this change was made in the first place? Is there any reason we don't leave the value where they were?

Reply:
answered above